hello.php

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/php
+<?php
+echo "Hello, world!\n";
```

Comment:
Instead of echoing `\n` explicitly, leaving an empty line at the end of the file would be easier.